Patches.py

```diff
@@ -2771,6 +2771,7 @@ def configure_dungeon_info(rom: Rom, world: World) -> None:
                 dungeon_rewards[codes.index(area.dungeon_name)] = boss_reward_index(location.item)
 
     dungeon_is_mq = [1 if world.dungeon_mq.get(c) else 0 for c in codes]
+    dungeon_precompleted = [1 if world.empty_dungeons[c].empty else 0 for c in codes]
```

Comment:
The existing names chosen for pre-completed dungeons seem to be misleading. They aren't empty dungeons, and empty as a property on the dungeon almost looks like a language feature to check if the index in an array is empty.

For clarity maybe these should be renamed to `precompleted_dungeons` and `precompleted`?

Reply:
I'm not sure the `.empty` attribute access is needed at all. I think `World.empty_dungeons` could just be a `dict[str, bool]`. I guess this PR is as good a time to change it as any, but it shouldn't be a blocker since it could also be handled in a separate PR. Regarding the name change, the current name matches the internal/plando name for the setting, so it should stay for now imo. Hopefully we can eventually accept #1839 so we can rename it.